src/huntsman/pocs/dome/musca.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, command_delay=1, max_status_attempts=3, *args, **kwargs):
 
         self._status = dict()
         self._status_delay = 5  # seconds
+        self._status_lines = 6
```

Comment:
We have a `Protocol.VALID_DEVICE` that specifies what keys are in the status call. We should use the length of that (+1) to query for the status so we don't have to track multiple things. See below.
```suggestion
```